### PR TITLE
CRM-20450 Update post upgrade message with correct list of modified t…

### PR DESCRIPTION
**Pull request by seamuslee001**
…emplates and fix closing tag which should be font

### CODE REVIEW

**Comment by seamuslee001** [comment]
ping @eileenmcnaughton @KarinG

File: xml/templates/message_templates/contribution_invoice_receipt_html.tpl [hunk 1]
```diff
@@ -156,7 +156,7 @@
                     {/if}
                     </font>
                   </td>
-                  <td style = "padding-left:34px;text-align:right;"><font size = "1">{$amountPaid|crmMoney:$currency}</font></td>
+                  <td style = "padding-left:34px;text-align:right;"><font size = "1">{$amount|crmMoney:$currency}</font></td>
```
Comment (eileenmcnaughton) [inline xml/templates/message_templates/contribution_invoice_receipt_html.tpl]: @seamuslee001 looks wrong

Comment (seamuslee001) [inline xml/templates/message_templates/contribution_invoice_receipt_html.tpl]: woops thanks

Comment (seamuslee001) [inline xml/templates/message_templates/contribution_invoice_receipt_html.tpl]: Fixed

**Comment by KarinG** [comment]
@seamuslee001 - Ah good eye on fonts...! I just copied that line from the all paid section of the tpl to make the invoice stop running off the screen in the non paid section. Never read it that closely. Thank you!

**Pull request merged**